### PR TITLE
Process review feedback

### PR DESCRIPTION
**Pull request by RonRademaker**
Hi, I processed the feedback I gave in the review

### CODE REVIEW

**Comment by rosstuck** [comment]
@tyx Any thoughts on this? :)

File: DependencyInjection/Compiler/CommandHandlerPass.php [hunk 1]
```diff
@@ -113,7 +113,7 @@ private function guardInvalidMiddlewares(ContainerBuilder $container, $busId)
         $busDefinition = $container->getDefinition('tactician.commandbus.'.$busId);
         foreach ($busDefinition->getArgument(0) as $middlewareReference) {
             if (false === $container->has($middlewareReference)) {
-                throw UnknownMiddleware::withId((string) $middlewareReference);
+                throw UnknownMiddlewareException::withId((string) $middlewareReference);
```
Comment (rosstuck) [inline DependencyInjection/Compiler/CommandHandlerPass.php:116]: This is more inline with the style we're using throughout the codebase.

**Comment by tyx** [comment]
Oh guys sorry ! Totally miss this PR ! I'm OK with all these changes :+1: 

I guess we should move them to master as long as my PR was merged ?

**Comment by rosstuck** [comment]
This has all been merged and the conflicts cleaned up. Feel free to close. 😄

**Pull request closed without merging**